test/src/main/java/org/apache/accumulo/test/ImportExportIT.java

```diff
@@ -151,7 +151,8 @@ public void testExportImportThenScan() throws Exception {
       log.info("Import dir B: {}", Arrays.toString(fs.listStatus(importDirB)));
 
       // Import the exported data into a new table
-      client.tableOperations().importTable(destTable, importDirDlm);
+      client.tableOperations().importTable(destTable,
+          Set.of(importDirA.toString(), importDirB.toString()));
```

Comment:
If you define this set earlier, you wouldn't need to manually concatenate them for the log message, because you could just rely on `Set.toString()`.